### ISSUE TITLE
URLs (both quoted and unquoted) break on parse

### ISSUE THREAD

**Issue by alexweinstein** [stolksdorf/CleanCSS#20]
This:
`background-image:url(http://example.com/image.png);`

is reformatted to this:
`background-image:url(http : //example.com/image.png);`

**Comment by stolksdorf** [comment]
Yes that will break because that's not standard syntax for using that css rule. You need to wrap your URL in quotes. [Check out the docs here](http://www.w3schools.com/cssref/pr_background-image.asp)

**Comment by alexweinstein** [comment]
I believe the quotes are optional: http://stackoverflow.com/questions/2168855/is-quoting-the-value-of-url-really-necessary

**Comment by stolksdorf** [comment]
Yup, you are right. I actually thought I was commenting on another project of mine. Sorry for the confusion.
